### PR TITLE
Mem 12 q xml

### PR DESCRIPTION
**Pull request by barricade86**

### CODE REVIEW

File: mem-12-q.xml [hunk 1]
```diff
@@ -85,7 +85,7 @@
       <column name="notes_de"></column>
       <column name="notes_fa"></column>
       <column name="notes_sv"></column>
-      <column name="notes_ru"></column>
+      <column name="notes_ru">Подобно слову {QaQ:v} данный глагол используется в контексте обозначения качества. Не используется для обозначения "не быть добрым"</column>
```
Comment (De7vID) [inline mem-12-q.xml:88]: Is this note necessary? Generally, the notes should be translations of each other.

File: mem-12-q.xml [hunk 2]
```diff
@@ -182,18 +182,19 @@
...
+      <column name="definition_ru">Здание, строение, структура, сооружение, конструкция</column>
       <column name="definition_zh_HK">房屋、樓</column>
       <column name="synonyms"></column>
       <column name="antonyms"></column>
       <column name="see_also">{HuDyar:n}, {chalqach:n}, {jem'IH:n}, {DuHmor:n}, {qappam:n}</column>
-      <column name="notes">The {yor:n} of a {qach:n} is its {beb:n}, and its {Hur:n}-facing walls are called {reD:n:1}. The other side of the {beb:n} (that is, the {'aqroS:n:1} of a {qach:n}) is called the {pa' beb:n}; the {qoD:n} face of a {reD:n:1} is referred to as {pa' reD:n}.
-
-For the {qoD:n} parts of a {qach:n}, see {pa':n:1}.</column>
+      <column name="notes">The {yor:n} of a {qach:n} is its {beb:n}, and its {Hur:n}-facing walls are called {reD:n:1}. The other side of the {beb:n} (that is, the {'aqroS:n:1} of a {qach:n}) is called the {pa' beb:n};
+the {qoD:n} face of a {reD:n:1} is referred to as {pa' reD:n}.</column>
```
Comment (De7vID) [inline mem-12-q.xml:191]: Bad edit?

File: mem-12-q.xml [hunk 3]
```diff
@@ -6395,7 +6456,9 @@ Maltz seemed reluctant to talk about what exactly this is.[2]</column>
 {qoghDu':n:nolink} can only mean "ears", while {qoghmey:n:nolink} is usually the plural of {qogh:n:1} and means "belts". The homophony between these two words is the basis of a slang expression, {qogh tuQmoHHa':sen}.[2]</column>
       <column name="notes_de"></column>
       <column name="notes_fa"></column>
-      <column name="notes_sv"></column>
+      <column name="notes_sv">Относится к внешнему лоскуту кожи, хрящу, и так далее. Внутренний орган слуха назван {teS:n}.
```
Comment (De7vID) [inline mem-12-q.xml:6459]: Should be "ru" not "sv".

File: mem-12-q.xml [hunk 4]
```diff
@@ -6145,7 +6203,7 @@ Maltz seemed reluctant to talk about what exactly this is.[2]</column>
       <column name="notes_de"></column>
       <column name="notes_fa"></column>
       <column name="notes_sv"></column>
-      <column name="notes_ru"></column>
+      <column name="notes_ru">Также подходит для обозначения процесса регистрации учётной записи {mab qI':sen@@mab:n, qI':v}.[2]</column>
```
Comment (De7vID) [inline mem-12-q.xml:6206]: This entry and note has changed. Please sync up with https://github.com/De7vID/klingon-assistant-data/tree/master (you're ~160 commits behind).

File: mem-12-q.xml (hunk 1 above)
Comment (barricade86) [reply mem-12-q.xml:88]: Yes. It is necessary. In entry `QaQ` we have similar note. I just made another one

**Pull request merged**